### PR TITLE
Feedback for ocp-4.1

### PR DESCRIPTION
**Pull request by danielhelfand**
This pull request contains feedback after going through the ocp-4.1 workshop. I will also include notes on issues I could not address in this pr.

### CODE REVIEW

**Comment by danielhelfand** [comment]
For the [logging section](https://github.com/openshift-labs/starter-guides/blob/ed9a5b6c0a19fc64451e8b97435fce4f8f4247cf/parksmap-logging.adoc), at the bottom of the page, the following instruction is present:

> If you click the "x" in the column for the container name, and, in the left bar, click "add" for kubernetes.pod_name, you’ll then see your old Pod's logs, too. Remember, we scaled them down before coming here, so you can see how the log system is keeping a historical record.

I could not the "x" as described above and could not complete this step. I'm wondering if it makes sense to remove it?

**Comment by danielhelfand** [comment]
For the [javascript-database section](https://github.com/openshift-labs/starter-guides/blob/7300b0322c9df2bf8a58afb075058cdc43d04161/nationalparks-javascript-databases.adoc), there is a section about loading the data that says I should expect the following:

```
Items inserted in database: 2740
```

What I get is:

```
504 Gateway Time-out
The server didn't respond in time.
```

However the data does load correctly.

**Comment by jankleinert** [comment]
> For the [logging section](https://github.com/openshift-labs/starter-guides/blob/ed9a5b6c0a19fc64451e8b97435fce4f8f4247cf/parksmap-logging.adoc), at the bottom of the page, the following instruction is present:
> 
> > If you click the "x" in the column for the container name, and, in the left bar, click "add" for kubernetes.pod_name, you’ll then see your old Pod's logs, too. Remember, we scaled them down before coming here, so you can see how the log system is keeping a historical record.
> 
> I could not not the "x" as described above and could not complete this step. I'm wondering if it makes sense to remove it?

Yes, can you remove it?

**Comment by danielhelfand** [comment]
@jankleinert Removed it. Thanks for verifying.

File: common-explore.adoc [hunk 1]
```diff
@@ -17,7 +16,7 @@ http://terminal-{{INFRA_PROJECT}}.{{ROUTER_ADDRESS}}
 You will need to log in with your OpenShift user and click on `Allow selected permissions`. You can then continue with the workshop.
 ====
 
-In order to log in, we will use the `oc` command and then specify the server that we
+In order to log in, we will use the `oc login` command and then specify the server that we
```
Comment (danielhelfand) [inline common-explore.adoc:19]: We will need a way here to dynamically pull in the new, separate url used for `oc` to log into an OpenShift cluster as opposed to the web console address.

**Comment by blues-man** [comment]
Thanks @danielhelfand we updated the lab to `ocp-4.5` can you check if everything have been fixed? In case you can open a new PR that help us fixing those in the new branch

Thanks!

**Pull request closed without merging**